common/landed_titles/gw_landed_titles.txt

```diff
@@ -0,0 +1,2031 @@
+#####TEMPORARY UNTIL TRIBUTARIES OF REDRIDGE PACK GET LAND UPON CB ENFORCED#####
+e_azeroth = {
```

Comment:
What `e_azeroth` doing in this file?

Reply:
Got it. You want `k_redridge` to be de-jure liege of duchies in the Redriege mountains, right? Move the `k_redridge` declaration to `wc_titular_titles` (we store titular titles there) and add the `de_jure_liege` in history of Redriege duchies.
<details>
<summary>Syntax example from vanilla</summary>
867.1.1={
	de_jure_liege = e_hre
}
</details>